frontend/src/Editor/QueryManager/QueryEditors/TooljetDatabase/operations.js

```diff
@@ -7,16 +7,16 @@ export const tooljetDbOperations = {
   perform,
 };
 
-async function perform(queryOptions, organizationId, currentState) {
-  switch (queryOptions.operation) {
+async function perform(dataQuery, currentState) {
+  switch (dataQuery.options.operation) {
     case 'list_rows':
-      return listRows(queryOptions, organizationId, currentState);
+      return listRows(dataQuery, currentState);
```

Comment:
@akshaysasidrn shouldn't we pass `dataQuery.options` here instead of just `dataQuery`? 🤔

Reply:
Ohh.. you passed it down in individual functions. Please ignore my bad. :(